uber/utils.py

```diff
@@ -224,6 +224,7 @@ def __init__(self, targets=(), amount=None, description=None):
 
         self.amount = amount or self.total_cost
         self.description = description or self.names
+        self.email = self.models[0].email if self.targets and self.models[0].email else ''
```

Comment:
This is the only file with relevant changes. The other stuff was from a (already-merged) PR.